submissions/reactjs/assignment2.MD

```diff
@@ -31,7 +31,7 @@
 | Achal Ladani                   | 2203031080229        |                   |             |
 | Harsh Choudhary                | 2203031080132        |                   |             |
 | Jeet Dhake                     | 2303031087082        |                   |             |
-| Vishang Lad                    | 2203031080035        |                   |             |
+| Vishang Lad                    | 2203031080035        |https://github.com/vishangl/React_Assignments/blob/main/Assignment%202/vite-project/src/components/Greet.jsx                   |https://github.com/vishangl/React_Assignments             |
```

Comment:
Please follow the repository structure mentioned in the problem statement document